grype/presenter/models/document.go

```diff
@@ -75,6 +81,7 @@ func NewDocument(packages []pkg.Package, context pkg.Context, matches match.Matc
 			Version:               version.FromBuild().Version,
 			Configuration:         appConfig,
 			VulnerabilityDBStatus: dbStatus,
+			Timestamp:             string(timestamp),
```

Comment:
If I could suggest an alternate method of testing: add a simple test that a valid timestamp is included from this `NewDocument` function, in `document_test.go`.

For the `presenter_test.go` tests, just add a `redact` function that removes the timestamp, something like this: 
```go
func redact(s string) string {
	return regexp.MustCompile(`"timestamp":\s*"[^"]+"`).
		ReplaceAllString(s, `"timestamp":""`)
}
```
and then the only change you need to make in the `presenter_test.go` is before the `assert.Equal`, add:
```go
actual = redact(actual)
```
... or something equivalent

Reply:
Yeah that makes way more sense